cocos/core/data/deserialize-dynamic.ts

```diff
@@ -263,7 +263,7 @@ function compileDeserializeNative (_self: _Deserializer, klass: CCClassConstruct
             if (prop === undefined) {
                 continue;
             }
-            if (!fastMode && typeof prop !== 'object') {
+            if (typeof prop !== 'object') {
```

Comment:
@holycanvas Please check if `!fastMode` not needed anymore.

Reply:
`compileDeserializeJit` does not have this check. I think it's better keep these uniform

Reply:
this check is needed. compileDeserializeJit is judged in the process of function generation, so there is no after generation, otherwise how to call jit